### DISCUSSION TITLE
Reserved words for directories

### DISCUSSION THREAD

**Discussion by joshuataylor** [retypeapp/retype#164] [Q&A] [answered]
Hi!

Really liking retype, it's super simple and sleek and looks great out of the box.

Is there a list of reserved words for directories, as I tried to make a folder called "categories" and it didn't show, which was super confusing and I thought I was doing something wrong (as this was my first folder).

Maybe a warning could show saying "warning, reserved folder name in use" or something, but this could also get annoying 🤔 .

**Comment by geoffreymcgill** [comment]
Thanks for the feedback. Glad to hear Retype caught your eye. 

Regarding reserved folder names... ya, good point. There might be a few (at the moment), but I would prefer there be none. 

The folders `blog`, [`tags`](https://retype.com/configuration/page/#tags), and [`categories`](https://retype.com/configuration/page/#category) are reserved right now. We need to document that right away, but it would be better if Retype just handled these folders differently.

Let me do some research and get back to you with a plan.

Thanks for making us aware of this issue.

**Reply by geoffreymcgill** [reply]
The new [Reserved words](https://retype.com/configuration/reserved_words/) documentation is now available.

**Comment by fabriciomurta** [comment, marked as answer]
I have drafted docs listing most (if not all) reserved resources from Retype, from what I could recollect. #168. As soon as we have some time to give it a proper review, we'll include it in the actual documentation. Will not be an annoying and perhaps useful message from retype console app, but at least a place we can point when other people have trouble with reserved paths.

**EDIT** The new [Reserved words](https://retype.com/configuration/reserved_words/) documentation is now available.
